redisvl/index/index.py

```diff
@@ -358,7 +358,7 @@ def _redis_client(self) -> Optional[redis.Redis]:
             with self._lock:
                 if self.__redis_client is None:
                     self.__redis_client = RedisConnectionFactory.get_redis_connection(
-                        url=self._redis_url,
+                        redis_url=self._redis_url,
```

Comment:
personally I think we should always use redis_url vs url because it's more specific and that's what we tend to say when we do it

Reply:
Agreed

Reply:
yup